src/resources/pdf/legibility.html

```diff
@@ -1,133 +1,92 @@
-  consequence of receiving this document. Unless stated otherwise, this document
-  represents only the views of the sender and not the views of the Queensland
-  Government.
+  Queensland Government takes all reasonable steps to ensure this document does not
+  contain malicious software, the Queensland Government does not accept responsibility
+  for the consequences if any person's computer inadvertently suffers any
+  disruption to services, loss of information, harm or is infected with a virus,
+  other malicious computer programme or code that may occur as a consequence of
+  receiving this document. Unless stated otherwise, this document represents
+  only the views of the sender and not the views of the Queensland Government.
 </small>
-<section>
-  <p>
-    <strong>Copyright</strong>
-    <span class="link-style">(https://www.qld.gov.au/legal/copyright/)</span>
-  </p>
-  <p>
-    <strong>Disclaimer</strong>
-    <span class="link-style">(https://www.qld.gov.au/legal/disclaimer/)</span>
-  </p>
-  <p>
-    <strong>Privacy</strong>
-    <span class="link-style">(https://www.qld.gov.au/legal/privacy/)</span>
-  </p>
-  <p>
-    <strong>Right to information</strong>
-    <span class="link-style"
-      >(https://www.qld.gov.au/right-to-information/)</span
-    >
-  </p>
-  <br />
-  <br />
-  <br />
-  <p>
-    <strong>© The State of Queensland 1995-2020</strong>
-  </p>
-  <p>
-    <strong>Queensland Government</strong>
-    <span class="link-style">(https://www.qld.gov.au/)</span>
-  </p>
+<br><br>
+<section class="notes-footer">
+  <small>
+    <p>
+      <strong>
+        Copyright
+        <a
+          href="https://www.qld.gov.au/legal/copyright/"
+          target="_blank"
+          rel="noopener"
+        >
+          (https://www.qld.gov.au/legal/copyright/)
+        </a>
+      </strong>
+    </p>
+    <p>
+      <strong>
+        Disclaimer
+        <a
+          href="https://www.qld.gov.au/legal/disclaimer/"
+          target="_blank"
+          rel="noopener"
+        >
+          (https://www.qld.gov.au/legal/disclaimer/)
+        </a>
+      </strong>
+    </p>
+    <p>
+      <strong>
+        Privacy
+        <a
+          href="https://www.qld.gov.au/legal/privacy/"
+          target="_blank"
+          rel="noopener"
+        >
+          (https://www.qld.gov.au/legal/privacy/)
+        </a>
+      </strong>
+    </p>
+    <p>
+      <strong>
+        Right to information
+        <a
+          href="https://www.qld.gov.au/right-to-information/"
+          target="_blank"
+          rel="noopener"
+        >
+          (https://www.qld.gov.au/right-to-information/)
+        </a>
+      </strong>
+    </p>
+    <p>
+      <strong>
+        <i class="fa fa-copyright" aria-hidden="true"></i> The State of
+        Queensland 1995-2020
+      </strong>
+    </p>
+    <p>
+      <strong>
+        Queensland Government
+        <a href="https://www.qld.gov.au/" target="_blank" rel="noopener">
```

Comment:
Not sure if the rel and that are necessary on PDF links, but you didn't add them so I guess no point in removing them.